src/Promitor.Core.Scraping/ResourceTypes/AzureSqlDatabaseScraper.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Azure.Management.Monitor.Fluent.Models;
+using Promitor.Core.Scraping.Configuration.Model.Metrics;
+using Promitor.Core.Scraping.Configuration.Model.Metrics.ResourceTypes;
+
+namespace Promitor.Core.Scraping.ResourceTypes
+{
+    /// <summary>
+    /// Scrapes an Azure SQL Database.
+    /// </summary>
+    public class AzureSqlDatabaseScraper : Scraper<AzureSqlDatabaseResourceDefinition>
+    {
+        private const string ResourceUriTemplate = "subscriptions/{0}/resourceGroups/{1}/providers/Microsoft.Sql/servers/{2}/databases/{3}";
+
+        /// <summary>
+        /// Initializes an instance of the <see cref="AzureSqlDatabaseScraper" /> class.
+        /// </summary>
+        /// <param name="scraperConfiguration">The scraper configuration</param>
+        public AzureSqlDatabaseScraper(ScraperConfiguration scraperConfiguration)
+            : base(scraperConfiguration)
+        {
+        }
+
+        protected override async Task<ScrapeResult> ScrapeResourceAsync(string subscriptionId, ScrapeDefinition<AzureResourceDefinition> scrapeDefinition, AzureSqlDatabaseResourceDefinition resource, AggregationType aggregationType, TimeSpan aggregationInterval)
+        {
+            var resourceUri = string.Format(
+                ResourceUriTemplate,
+                AzureMetadata.SubscriptionId,
+                scrapeDefinition.ResourceGroupName,
+                resource.ServerName,
+                resource.DatabaseName);
+
+            var metricName = scrapeDefinition.AzureMetricConfiguration.MetricName;
+            var foundMetricValue = await AzureMonitorClient.QueryMetricAsync(metricName, aggregationType, aggregationInterval, resourceUri);
+
+            var labels = new Dictionary<string, string>
```

Comment:
Can you update the docs here please https://promitor.io/metrics/labels

Reply:
Done.